src/test/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
 spring.jpa.hibernate.ddl-auto=create
```

Comment:
Have a look at what I have done for this file in the CVR retrieval branch.

Reply:
Yes that's a huge improvement, thanks. I won't pull it into this branch, but definitely main should be that.